packages/app/src/specs/SpecListGitInfo.vue

```diff
@@ -1,24 +1,35 @@
 <template>
-  <div class="h-full grid gap-9px grid-cols-[16px,auto] git-info-row items-center">
-    <span
-      class="flex w-16px items-center justify-center"
-      :data-cy="`git-status-${props.gql.statusType}`"
+  <div
+    class="h-full grid gap-9px grid-cols-[16px,auto] git-info-row items-center"
+    data-cy="git-info-row"
+  >
+    <Tooltip
+      placement="top"
+      class="h-full grid items-center"
+      :disabled="!classes.showTooltip"
     >
-      <span
-        v-if="props.gql.statusType !== 'unmodified'"
-        class="rounded-full border-1 min-w-6px min-h-6px max-w-6px max-h-6px"
-        :class="classes?.indicator"
+      <component
+        :is="classes.icon"
+        :class="classes.iconClasses"
       />
-      <IconGithub
-        v-else
-        class="min-w-16px min-h-16px max-w-16px max-h-16px icon-dark-gray-300 group-hocus:icon-dark-indigo-500"
-      />
-    </span>
+      <template
+        v-if="classes.showTooltip"
+        #popper
+      >
+        <div>
+          <p class="max-w-sm text-sm truncate overflow-hidden">
+            {{ props.gql.subject }}
+          </p>
+          <p class="text-xs">
+            {{ gitTooltipSubtext }}
+          </p>
+        </div>
+      </template>
+    </Tooltip>
```

Comment:
I think changing this is out of scope in the near term, but we need to come up with a strategy for making this and future tooltip content available to users that can't or don't want to use mouse input. 

The tooltip target seems to catch focus and open the tooltip when it receives focus, but that conflicts with our desire to update the Specs List to use standard arrow keys for directory navigation: https://cypress-io.atlassian.net/browse/UNIFY-864. Given the scope of the additions being planned, though, we may need to change our desired approach.

Reply:
Interesting. I guess we need to address this separately.